activerecord/lib/active_record/migration.rb

```diff
@@ -885,13 +885,14 @@ def method_missing(method, *arguments, &block)
 
     def copy(destination, sources, options = {})
       copied = []
+      schema_migration = options[:schema_migration] || ActiveRecord::Base.connection.schema_migration
```

Comment:
Is this `options[:schema_migration]` will be used in the future? It is not used yet in this PR.